src/components/media-views.js

```diff
@@ -437,6 +441,10 @@ AFRAME.registerComponent("media-video", {
     entity.addEventListener("image-loaded", this.onSnapImageLoaded, ONCE_TRUE);
   },
 
+  chestImage(){
+    this.el.sceneEl.querySelector("#player-rig").querySelector(".image").setAttribute("material", {src:this.data.src, color:"white"});
```

Comment:
Would be nice if we could contain these chest image manipulations in a single place, instead of having `media-views` know about player-rig details. I would suggest emitting an event here instead, something like:
```javascript
this.el.sceneEl.emit("change_emoji", { src: this.data.src });
```